src/internet_identity/src/storage/storable_persistent_state.rs

```diff
@@ -53,8 +55,8 @@ impl From<PersistentState> for StorablePersistentState {
             active_anchor_stats: s.active_anchor_stats,
             domain_active_anchor_stats: s.domain_active_anchor_stats,
             active_authn_method_stats: s.active_authn_method_stats,
-            latest_delegation_origins: s.latest_delegation_origins,
-            max_num_latest_delegation_origins: s.max_num_latest_delegation_origins,
+            latest_delegation_origins: Default::default(),
```

Comment:
Do we want the same comment `unused, kept for stable memory compatibility` here and below?